build-posix.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+echo "Installing .NET CLI..."
+source "./evergreen/dotnet-install.sh" --channel LTS
+dotnet --info
+dotnet new tool-manifest --force
+dotnet tool install Cake.Tool
+dotnet cake --bootstrap
+dotnet cake "$@"
```

Comment:
Using Cake.Tool to actually run the build. The Windows script will change to this as well.